db/migrate/20211211180708_create_join_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTable < ActiveRecord::Migration[6.1]
+  def change
+    create_join_table :users, :posts, column_options: { null: false }, table_name: :likes do |t|
+      t.index :user_id, unique: true
+      t.index :post_id, unique: true
+    end
```

Comment:
- [ ] Here as well. As per the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the column name should be `author` instead of `user` in the `likes` table. Please fix it.

```ruby
create_table :likes do |t|
    ...
    t.references :author, foreign_key: { to_table: 'users' }
  end
```